src/init.cpp

```diff
@@ -713,16 +713,16 @@ static void ThreadImport(std::vector<fs::path> vImportFiles)
  */
 static bool InitSanityCheck()
 {
-    if(!ECC_InitSanityCheck()) {
-        InitError("Elliptic curve cryptography sanity check failure. Aborting.");
+    if (!ECC_InitSanityCheck()) {
+        InitError(_("Elliptic curve cryptography sanity check failure. Aborting.", true));
```

Comment:
I don't understand why this should be translated. This should never fail and if it fails, I bet you are better off with the english error message?

Reply:
Agree, I feel we had this discussion already. Please don't translate extremely rare or technical errors.

Reply:
Fixed.

btw, this PR aims that a translated message:
- is written to debug log in _English_
- is printed to `stderr` in _English_
- is shown in GUI in both languages

Therefore, english error message is always available even for translated messages.